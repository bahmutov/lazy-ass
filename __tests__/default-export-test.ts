import la from '../src'

describe('lazy-ass as default export', () => {
  it('does not throw if condition is true', () => {
    la(true)
  })

  it('does not throw when there is a message', () => {
    la(true, 'text', 42, 'more text')
  })

  it('throws nice message', () => {
    expect(() => {
      la(false, 'text', 42, 'more text')
    }).toThrowErrorMatchingSnapshot()
  })
})
