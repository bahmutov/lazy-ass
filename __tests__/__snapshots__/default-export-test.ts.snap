// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lazy-ass as default export throws nice message 1`] = `"text 42 more text"`;
